src/realtime/websocket/utils/extract-note-id-from-request-url.ts

```diff
@@ -20,8 +20,8 @@ export function extractNoteIdFromRequestUrl(request: IncomingMessage): string {
   // The example.org domain should be safe to use according to RFC 6761 §6.5.
   const url = new URL(request.url, 'https://example.org');
   const noteId = url.searchParams.get('noteId');
-  if (noteId === null) {
-    throw new Error("Path doesn't contain parameter noteId");
+  if (noteId === null || noteId === '') {
+    throw new Error(`Path doesn't contain parameter noteId: ${request.url}`);
```

Comment:
Are we sure about using raw unsanitized user input (the URL) in an error message? This sounds like a possibility for unwanted injections...